libcli.c

```diff
@@ -2292,18 +2328,21 @@ int cli_int_enter_buildmode(struct cli_def *cli, struct cli_pipeline_stage *stag
                             optarg->privilege, cli->mode, optarg->help, optarg->get_completions, optarg->validator,
                             NULL);
       } else {
-        return CLI_BUILDMODE_ERROR;
+          rc = CLI_BUILDMODE_ERROR;
+	  goto out;
```

Comment:
Indenting is inconsistent here (goto is back 1 character) and the next few.

Reply:
<sigh> tab vs space issue - I normally have my editors set up to turn literal tabs into spaces (unless I'm editing makefiles), this edit was made w/o that setting and the editor kept the tab.  Fixed.  Too much time in python land over the past years.....